sdks/wasm/runtime-tests.js

```diff
@@ -154,7 +158,40 @@ var Module = {
 			var f = Module.cwrap ('mono_wasm_enable_on_demand_gc', 'void', []);
 			f ();
 		}
-
+		if (enable_zoneinfo) {
```

Comment:
This is a test file, how is this done in an app ?

Reply:
In the description of the PR there is a section on Zone Info data and support files.  This is loaded in the browser.  The code in the zone info data files contains code to load the zoneinfo data into the VFS.  Basically the same code as the test file. 

The package files generated do not contain code to load via node.js or code executed from JS engines.